superset/assets/javascripts/SqlLab/components/SqlEditorLeftBar.jsx

```diff
@@ -1,9 +1,12 @@
+require('es6-promise').polyfill();
+require('isomorphic-fetch');
 const $ = window.$ = require('jquery');
 import React from 'react';
 import Select from 'react-select';
 import { Label, Button } from 'react-bootstrap';
 import TableElement from './TableElement';
 import AsyncSelect from '../../components/AsyncSelect';
+import fetch from 'isomorphic-fetch';
```

Comment:
since we are still using/importing jquery in this file, i think it makes sense to continue using `$.get` rather than importing `es-promise` and `isomorphic-fetch` as well. once we can get rid of the jquery dependency i think it makes sense to switch to using `isomorphic-fetch` everywhere.